lib/dotcom/trip_plan/input_form.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dotcom.TripPlan.InputForm do
   use TypedEctoSchema
   import Ecto.Changeset
 
-  @valid_modes [:commuter_rail, :subway, :bus, :ferry]
+  alias OpenTripPlannerClient.PlanParams
+
+  @valid_modes [:RAIL, :SUBWAY, :BUS, :FERRY]
```

Comment:
Should alphabetize these. And maybe even use the sigil `~w(BUS FERRY RAIL SUBWAY)a`. Not sure if that makes it easier or harder to read.